elife/golang-go.sls

```diff
@@ -0,0 +1,3 @@
+golang-go:
```

Comment:
probably naming this file `go.sls` is enough (may contain configuration like GOPATH in addition to the package)